Testscripts/Linux/validate-da.sh

```diff
@@ -81,30 +94,33 @@ function verify_expected_file() {
 }
 
 function verify_da_pid() {
-    [ ! -f $da_pid_file ] && fail_test "$da_pid_file does not exist."
+    if [ ! -f $da_pid_file ]; then
+        fail_test "$da_pid_file does not exist."
+    fi
 
-    if x=$(sed -n '/^[1-9][0-9]*$/p;q' $da_pid_file); then
+    if ! x=$(sed -n "/^[1-9][0-9]*$/p;q" $da_pid_file); then
+        fail_test "PID not found in DA_PID"
+    else
         if strings /proc/$x/cmdline | fgrep -q  -e "microsoft-dependency-agent-manager"; then
```

Comment:
Because `fail_test` causes a short-circuit exit from the function, the `if strings ...` statement doesn't need to be in an else clause. Continue the pattern started with `if [ ! -f $da_pid_file ]`, above.

Then, continue the pattern with the strings | fgrep assertion,

The final result will have a consistent structure:

if _assertion 1 fails_; then
    fail_test ...
fi

if _assertion 2 fails_; then
    fail_test ...
fi

if _assertion 3 fails_; then
    fail_test ...
fi

...

LogMsg _success_

Reply:
Done